cassandane/Makefile

```diff
@@ -60,7 +60,10 @@ MODULES := $(shell find . -type f -name '*.pm' | sort)
 # checks to proceed when no Cyrus install exists
 ifndef NOCYRUS
     SCRIPTS += $(shell find . -type f -name 'annotator.pl')
-    CYRUS_PERL_PATHS := $(shell $(PERL) utils/cyrus-perl-paths.pl)
+    ifdef CASSANDANE_INI
+	CYRUS_PERL_PATHS_ARGS := --config $(CASSANDANE_INI)
+    endif
+    CYRUS_PERL_PATHS := $(shell $(PERL) utils/cyrus-perl-paths.pl $(CYRUS_PERL_PATHS_ARGS))
```

Comment:
If this behaviour is going to be based around an environment variable being set anyway, would it not be simpler to just have Cassini itself look for a cassandane.ini path defined in an environment variable, rather than all this work to pass it down?  And then anything that uses Cassini (such as cyrus-perl-paths.pl, but also all of cassandane) doesn't need any special handling of its own, it can just ask for a Cassini instance as usual and get something coherent.

Cassini can already read any or all of its config entirely from the environment -- with potentially no cassandane.ini required at all (see `sub val` for the main implementation, and the end of `sub new` for the handling when no cassandane.ini file was found, where a `CASSINI_CASSANDANE_ALLOW_NOINIFILE` can suppress the error).  The only configuration detail it can't currently read from the environment is a path to a cassandane.ini file, and that seems trivial enough to add?  (Probably in `sub new` somewhere, but I can't just add a suggestion because a review won't let me comment on lines that weren't edited already.  I would probably call this environment variable something like `CASSINI_FILENAME`, for consistency with the other environment variables it looks for.)